__tests__/app.test.js

```diff
@@ -134,6 +134,59 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("200: responds with 200 status code", () => {
```

Comment:
This doesn't need to be two tests, in your next test you are asserting the status code is also 200